### ISSUE TITLE
Some questions about prepare_data and evaluation code

### ISSUE THREAD

**Issue by kangya998** [rsy6318/PUGeoNet_pytorch#4]
Hi, Is this the official version of the PUGeo-Net open source code, and is the dataset creation and evaluation code the one used in the paper? I am looking forward to your reply.

**Comment by rsy6318** [comment]
The official version is Tensorflow, the link is at the beginning of README.md

**Comment by kangya998** [comment]
> The official version is Tensorflow, the link is at the beginning of README.md
Thank you for your reply, I would like to make my own dataset using the method in PUGeo-Net, but the official version does not provide a method to generate training and test datasets, is your dataset generation method quoted from the author?

**Comment by rsy6318** [comment]
Yes, we use the same method to generate the dataset, and you can use my code to generate your own dataset directly.

**Comment by kangya998** [comment]
Okk, thanks for your help, I also saw your issue in the Neural Points repostory, are you still following up? I still doubt  about its implementation in the code, maybe we can discuss it with each other.

**Comment by rsy6318** [comment]
Neural Points are also in a patch manner, but its setting is different from PUGEO, i.e., PUGEO use 5000 points as basis, while Neural Points use 10000 points.

**Comment by rsy6318** [comment]
You can use Neural Points code in our main file directly to train your own dataset.

**Comment by kangya998** [comment]
Yes, I have tried using the code in your repository to create dataset with the mesh in PUGeo, and using the code from Neural Points to train without augmentation,  test and ealuation with PUGCN[https://github.com/guochengqian/PU-GCN/blob/master/evaluate.py], There is a large gap between the results and the paper. Have you tried pre-trained model in neural points?

**Comment by rsy6318** [comment]
We used different normalization for point cloud, so it is OK for the different results.

**Comment by rsy6318** [comment]
.

**Comment by Taousst** [comment]
Hi, are you working on point cloud sampling as well? Can you add a friend to discuss it
> Okk, thanks for your help, I also saw your issue in the Neural Points repostory, are you still following up? I still doubt about its implementation in the code, maybe we can discuss it with each other.

**Comment by Taousst** [comment]
> Okk, thanks for your help, I also saw your issue in the Neural Points repostory, are you still following up? I still doubt about its implementation in the code, maybe we can discuss it with each other.

Hi, are you working on point cloud sampling as well? Can you add a friend to discuss it
